wallet/src/Pos/Wallet/Web/Methods/Misc.hs

```diff
@@ -84,3 +96,10 @@ testResetAll = deleteAllKeys >> testReset
     deleteAllKeys = do
         keyNum <- length <$> getSecretKeys
         replicateM_ keyNum $ deleteSecretKey 0
+
+-- | Build Swagger-specification from 'walletApi'.
+swaggerSpec :: Swagger
+swaggerSpec = toCustomSwagger walletApi
+    & info . title       .~ "Cardano SL Wallet Web API"
+    & info . version     .~ toText (showVersion CSL.version)
+    & info . description ?~ "This is an API for Cardano SL wallet."
```

Comment:
As far as I understand, it's taken from executable. Could you explain why did you remove `host` field?

Reply:
>  If the host is not included, the host serving the documentation is to be used (including the port).

^ See https://hackage.haskell.org/package/swagger2-2.1.4.1/docs/Data-Swagger.html#g:8

Reply:
Aha, then it makes sense